de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/ArchiveOutgoingReferencePointNegotiation.java

```diff
@@ -30,19 +29,19 @@
 import de.fu_berlin.inf.dpp.synchronize.StartHandle;
 
 /**
- * Implementation of {@link AbstractOutgoingProjectNegotiation} utilizing a
- * transferred zip archive to exchange differences in the project files.
+ * Implementation of {@link AbstractOutgoingReferencePointNegotiation} utilizing
+ * a transferred zip archive to exchange differences in the project files.
  */
-public class ArchiveOutgoingProjectNegotiation extends
-    AbstractOutgoingProjectNegotiation {
+public class ArchiveOutgoingReferencePointNegotiation extends
+    AbstractOutgoingReferencePointNegotiation {
 
     private static final Logger LOG = Logger
-        .getLogger(ArchiveOutgoingProjectNegotiation.class);
+        .getLogger(ArchiveOutgoingReferencePointNegotiation.class);
     private File zipArchive = null;
 
-    public ArchiveOutgoingProjectNegotiation( //
+    public ArchiveOutgoingReferencePointNegotiation( //
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 10 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWaWrfc1sSPV6ZXI_B9P&open=AWaWrfc1sSPV6ZXI_B9P)

[ISSUE_KEY]: AWaWrfc1sSPV6ZXI_B9P